### PR TITLE
open bcf from platform

### PR DESCRIPTION
**Pull request by Amoki**
https://platform-dev-open-bcf.bimdata.io

null

### CODE REVIEW

File: src/views/model-viewer/ModelViewer.vue [hunk 1]
```diff
@@ -107,7 +107,7 @@ export default {
           accessToken: accessToken.value,
           cloudId: spaceID,
           projectId: projectID,
-          ifcIds: modelIDs,
+          modelIds: modelIDs,
```
Comment (Kurtil) [inline src/views/model-viewer/ModelViewer.vue:110]: 🎉

File: src/components/specific/bcf/bcf-topics-list/open-topic-issue/OpenTopicIssue.vue [hunk 2]
```diff
@@ -316,6 +378,24 @@ export default {
       }
     });
 
+    const modelIDs = computed(() => {
+      let ids = [];
+      if (props.bcfTopic.ifcs && props.bcfTopic.ifcs.length) {
+        ids = props.bcfTopic.ifcs;
```
Comment (Kurtil) [inline src/components/specific/bcf/bcf-topics-list/open-topic-issue/OpenTopicIssue.vue:356]:
```javascript
ids.push(...props.bcfTopic.ifcs);
```

File: src/components/specific/bcf/bcf-topics-list/open-topic-issue/OpenTopicIssue.vue [hunk 3]
```diff
@@ -316,6 +378,24 @@ export default {
       }
     });
 
+    const modelIDs = computed(() => {
+      let ids = [];
+      if (props.bcfTopic.ifcs && props.bcfTopic.ifcs.length) {
```
Comment (Kurtil) [inline src/components/specific/bcf/bcf-topics-list/open-topic-issue/OpenTopicIssue.vue]:
```javascript
props.bcfTopic.ifcs?.length
```
:P

**Review by Kurtil — APPROVED** [review]

**Review by pvilalta — APPROVED** [review]

**Pull request merged**